custom/icds_reports/views.py

```diff
@@ -209,6 +209,9 @@ def get_context_data(self, **kwargs):
         kwargs.update(self.kwargs)
         kwargs['location_hierarchy'] = location_hierarchy_config(self.domain)
         kwargs['user_location_id'] = self.couch_user.get_location_id(self.domain)
+        kwargs['all_user_location_id'] = list(self.request.couch_user.get_sql_locations(
+            self.kwargs['domain']
+        ).location_ids())
```

Comment:
@czue  No this wasn't on the other branch. I added this now because when I resolved conflicts and I tested my changes, I found one more use case where my solution doesn't work. (I know I should add this to new commit).

Reply:
ok, makes sense - thanks for clarifying!